src/main/java/org/broadinstitute/hellbender/tools/exome/orientationbiasvariantfilter/OrientationBiasUtils.java

```diff
@@ -78,26 +79,23 @@ public static String getGenotypeString(final Genotype g, final String fieldName)
 
     /** Complement of the artifact mode is NOT considered.
      *
-     *  Must be diploid genotype!
-     *
      *  The genotype can contain indels, be a reference genotype, etc.  That is all considered.
      *
      * @param g Never {@code null}
      * @param transition Never {@code null}
-     * @return whether the given genotype falls into the given artifact mode.  Complement of the artifact mode is NOT considered.
+     * @return whether *any* alleles in the given genotype falls into the given artifact mode (ref/alt).
+     *   Complement of the artifact mode is NOT considered.
      */
     public static boolean isGenotypeInTransition(final Genotype g, final Transition transition) {
         Utils.nonNull(g, "Genotype cannot be null");
         Utils.nonNull(transition, "Artifact mode cannot be null");
 
-        if (g.getAlleles().size() != 2) {
-            throw new UserException.BadInput("Cannot run this method on non-diploid genotypes.");
-        }
-        return g.getAllele(0).getDisplayString().equals(String.valueOf(transition.ref())) &&
-                g.getAllele(1).getDisplayString().equals(String.valueOf(transition.call()));
+        return IntStream.range(1,g.getAlleles().size()).anyMatch(i -> g.getAllele(0).getDisplayString().equals(String.valueOf(transition.ref())) &&
```

Comment:
This should be doable with `Allele::basesMatch` without converting the allele and the transition base to `String`s.